src/App.vue

```diff
@@ -4,7 +4,7 @@
 import Blockly from 'blockly/core';
 import { ref } from "vue";
 import BlocklyComponent from "./components/BlocklyComponent.vue";
-import { toolboxString } from './toolbox'
+import toolbox from './toolbox'
 import { javascriptGenerator } from "blockly/javascript";
```

Comment:
Could we normalize the way we import, or it is required that we put some specific ones into brackets? If so, why?

Reply:
the brakets extract the value from the import, there technically pointless but they minimize the usage of module.thing.thing.thing when using larger modules with lots of functions and values and various different places

Reply:
> the brackets ⋯ are technically **pointless** but **they minimize the usage of module.thing.thing.thing when using larger modules with lots of functions and values and various different places**

Then, they are not really pointless

Reply:
yeah
i removed them from toolbox value because for that they litteraly where pointless